pkg/translate/translate.go

```diff
@@ -97,6 +101,9 @@ func (t *TranslateServiceServer) DownloadTranslationFile(
 	ctx context.Context,
 	req *pb.DownloadTranslationFileRequest,
 ) (*pb.DownloadTranslationFileResponse, error) {
+	span := trace.SpanFromContext(ctx)
+	defer span.End()
```

Comment:
The result looks like this:
<img width="1641" alt="image" src="https://user-images.githubusercontent.com/69954124/219682461-865a36c1-dc1f-4bd9-98a2-ad6aea0b6b4b.png">

Reply:
Modified context is accessible via `trace.ContextWithSpan` e.g. 
```go
newCtx := trace.ContextWithSpan(ctx,span)
```